src/topology/metric_space/basic.lean

```diff
@@ -677,6 +681,13 @@ instance metric_space.to_separated : separated_space α :=
 separated_def.2 $ λ x y h, eq_of_forall_dist_le $
   λ ε ε0, le_of_lt (h _ (dist_mem_uniformity ε0))
 
+lemma metric.discrete_of_fintype {X : Type*} [metric_space X] [fintype X] : discrete_topology X :=
+discrete_of_t1_of_finite
+
+lemma metric.discrete_of_finite {X : Type*} [metric_space X] {s : finset X} :
+  discrete_topology {a : X // a ∈ s} :=
+discrete_of_t1_of_finite
```

Comment:
I'm not sure that we should have these lemmas. Why not use `discrete_of_t1_of_finite` directly?

Reply:
Removed these two lemmas!